Description: Soundtouch >= 1.5 uses soundtouch.pc and not soundtouch-1.4.pc
Author: Benjamin Drung <bdrung@debian.org>
Applied-Upstream: 1.3.13

--- a/m4/audacity_checklib_libsoundtouch.m4
+++ b/m4/audacity_checklib_libsoundtouch.m4
@@ -31,6 +31,11 @@
                      soundtouch_available_system="yes",
                      soundtouch_available_system="no")
    fi
+   if test "x$soundtouch_available_system" = "xno" ; then
+      PKG_CHECK_MODULES(SOUNDTOUCH, soundtouch >= 1.3.0,
+                     soundtouch_available_system="yes",
+                     soundtouch_available_system="no")
+   fi
 
    if test "x$soundtouch_available_system" = "xyes" ; then
       LIBSOUNDTOUCH_SYSTEM_AVAILABLE="yes"
